tests/components/electric_kiwi/test_sensor.py

```diff
@@ -65,6 +69,50 @@ async def test_hop_sensors(
     assert state.attributes.get(ATTR_DEVICE_CLASS) == SensorDeviceClass.TIMESTAMP
 
 
+@pytest.mark.parametrize(
+    ("sensor", "sensor_state"),
+    [
+        ("sensor.total_running_balance", "184.09"),
+        ("sensor.total_current_balance", "-102.22"),
+        ("sensor.next_billing_date", "2020-11-03T00:00:00"),
+        ("sensor.hour_of_power_savings", "3.5"),
+    ],
+)
+async def test_account_sensors(
+    hass: HomeAssistant,
+    config_entry: MockConfigEntry,
+    ek_api: YieldFixture,
+    ek_auth: YieldFixture,
+    entity_registry: EntityRegistry,
+    component_setup: ComponentSetup,
+    sensor: str,
+    sensor_state: str,
+) -> None:
+    """Test Account sensors for the Electric Kiwi integration."""
+
+    await component_setup()
+    assert config_entry.state is ConfigEntryState.LOADED
+
+    entity = entity_registry.async_get(sensor)
+    assert entity
+
+    api = ek_api(Mock())
+    account_data = await api.get_account_balance()
+    assert account_data
+
+    AccountSensors = list(
+        filter(lambda x: entity.unique_id.endswith(x.key), ACCOUNT_SENSOR_TYPES)
+    )
+    AccountSensor: ElectricKiwiAccountSensorEntityDescription = AccountSensors[0]
+
+    state = hass.states.get(sensor)
+    assert state
+    assert state.state == sensor_state
+    assert state.attributes.get(ATTR_ATTRIBUTION) == ATTRIBUTION
+    assert state.attributes.get(ATTR_DEVICE_CLASS) == AccountSensor.device_class
+    assert state.attributes.get(ATTR_STATE_CLASS) == AccountSensor.state_class
```

Comment:
Add this to the parameters instead
```suggestion
    assert state.attributes.get(ATTR_DEVICE_CLASS) == .device_class
    assert state.attributes.get(ATTR_STATE_CLASS) == state_class
```